### ISSUE TITLE
Splenic infarction

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#7697]
**Preferred term label:**

Splenic infarction

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 23281181

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hereditary persistence of fetal hemoglobin-sickle cell disease syndrome, ORPHA:251380

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding term as HP_0034336

Def: Ischemia and necrosis of part or all of the spleen resulting from compromise of blood supply resulting from arterial or venous occlusion.
Comment: Splenic infarction may be global or involve a segment of the spleen depending on which vessel is occluded. Typical presentation includes left sided abdominal pain in a person with an underlying hematologic disorder, hypercoagulable state, blood-borne malignancy, blunt abdominal trauma, or embolic illness.

PMID:23281181
PMID:28613652
